src/router.ts

```diff
@@ -80,7 +81,28 @@ router.post("/api/v1/bookshelf", async (req, res) => {
       bookId: bookInstance.id,
     });
 
-    res.sendStatus(204);
+    if (shelfType === "already_read") {
+      const missingParams = [];
+      if (!review) {
+        missingParams.push("review");
+      }
+      if (!rating) {
+        missingParams.push("rating");
+      }
+      if (missingParams.length) {
+        res.status(400).json({
+          message: `Error. You must provide ${missingParams} for already read books`,
+        });
+        return;
+      }
+      Review.create({
```

Comment:
```suggestion
      await Review.create({
```

Be sure to `await` all promises/async function calls, unless you have a specific reason not to. Otherwise, any errors thrown inside the async function (`Review.create()` in this case) will not get caught by any `try/catch` wrapping it 💀